### PR TITLE
ISSUES-10 Add Enum8 Enum16  UUID DataType

### PR DESCRIPTION
**Pull request by sundy-li**
ISSUES-10 Add Enum DataType #10

### CODE REVIEW

**Comment by sundy-li** [comment]
almost done

File: src/main/java/org/houseflys/jdbc/type/ColumnFactory.java [hunk 1]
```diff
@@ -41,8 +49,8 @@ public static ColumnCreator getCreator(String type) throws SQLException {
     public static ParseResult parseTypeName(String type, int pos) throws SQLException {
         Matcher matcher = SIMPLE_TYPE_REGEX.matcher(type);
         return matcher.find(pos) && matcher.start() == pos &&
-            (matcher.end() == type.length() || type.charAt(matcher.end()) != '(') ?
-            parseSimpleTypeName(matcher.group(1), matcher.end()) : parseComplexTypeName(type, pos);
+                   (matcher.end() == type.length() || type.charAt(matcher.end()) != '(') ?
```
Comment (zhang2014) [inline src/main/java/org/houseflys/jdbc/type/ColumnFactory.java]: Maybe we should have a consistent code style

**Comment by zhang2014** [comment]
For Enum DataType, Maybe we should support both the getString and getInt of the ResultSet

File: src/main/java/org/houseflys/jdbc/type/ColumnFactory.java [hunk 2]
```diff
@@ -82,9 +92,12 @@ private static ParseResult parseComplexTypeName(String type, int pos) throws SQL
         creators.put("UInt32", new Int32ColumnCreator());
         creators.put("UInt64", new Int64ColumnCreator());
         creators.put("String", new StringColumnCreator());
+        creators.put("Enum8", new StringColumnCreator());
```
Comment (zhang2014) [inline src/main/java/org/houseflys/jdbc/type/ColumnFactory.java]: Better is to use EnumColumnCerator

**Pull request merged**